invoices_app/filters.py

```diff
@@ -1,48 +1,78 @@
-from django.forms import Select
-from django_filters import ChoiceFilter, DateFromToRangeFilter, FilterSet
-from django_filters.widgets import SuffixedMultiWidget
+from django.forms import NumberInput, Select, CheckboxSelectMultiple
+from django_filters import (
+    DateFromToRangeFilter,
+    FilterSet,
+    ModelChoiceFilter,
+    MultipleChoiceFilter,
+    RangeFilter,
+)
+from django_filters.widgets import RangeWidget, SuffixedMultiWidget
 
 from bootstrap_datepicker_plus import DatePickerInput
 
+from supplier_app.models import TaxPayer
 from invoices_app import INVOICE_STATUS
 from invoices_app.models import Invoice
 
 
-class CustomRangeWidget(SuffixedMultiWidget):
+class NumericRangeWidget(RangeWidget):
     template_name = 'django_filters/widgets/multiwidget.html'
     suffixes = ['min', 'max']
 
+    def __init__(self, attrs=None):
+        widgets = (
+            NumberInput(attrs={'class': 'form-control', 'placeholder': 'Min'}),
+            NumberInput(attrs={'class': 'form-control', 'placeholder': 'Max'})
+        )
+        super(SuffixedMultiWidget, self).__init__(widgets, attrs)
+
+
+class DateRangeWidget(RangeWidget):
+    template_name = 'django_filters/widgets/multiwidget.html'
+    suffixes = ['after', 'before']
+
     def __init__(self, attrs=None):
         widgets = (
             DatePickerInput(options={
                     "format": "MM/DD/YYYY",
```

Comment:
its a good practice to have this kind of string as constants, in order to reutilize them and also to avoid typos ;)